src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,31 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.exceptions.DataNotFoundException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import org.hibernate.SessionFactory;
+
+public class ActorServiceImpl extends AbstractService implements ActorService {
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(SessionFactory sessionFactory) {
+        super();
+        this.actorDao = new ActorDaoImpl(sessionFactory);
```

Comment:
Consider using dependency injection for the `ActorDao` instead of instantiating it directly in the constructor. This will improve the flexibility and testability of the code.